### PR TITLE
Change numbering process

### PR DESCRIPTION
**Pull request by franciscoaguirre**
I started with the idea of numbering accepted RFCs linearly, 0001, 0002, 0003, etc.
This meant you would add the number after merging the RFC.

Most other processes use the PR number which is unique anyway.
It makes it simpler since you can know the name of the RFC before it's accepted and it doesn't depend on other RFCs getting accepted first.

### CODE REVIEW

**Comment by KiChjang** [comment]
We should also renumber 0001 as 0032, which is the PR that introduced the process RFC.

**Comment by franciscoaguirre** [comment]
@KiChjang updated the file name

File: proposals/0032-process.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 Title: RFC process
-Number: 1
+Number: 32
 Status: Enacted
-Version: 0
+Version: 1
```
Comment (KiChjang) [inline proposals/0032-process.md:5]:
```suggestion
Version: N/A
```

since it doesn't follow any XCM version.

**Comment by KiChjang** [comment]
Actually, we should really explain what the `version` field on the header is about. Originally I thought it was used to say which XCM version that the RFC is intended to target, but on the set asset claimer RFC, you mentioned that it's the version of the RFC. Could you please clarify its meaning?

**Review by vstam1 — APPROVED** [review]
Approved when final comment of Keith is addressed

**Comment by KiChjang** [comment]
Still needs an explanation on the RFC about versioning.

**Comment by muharem** [comment]
> Actually, we should really explain what the `version` field on the header is about. Originally I thought it was used to say which XCM version that the RFC is intended to target, but on the set asset claimer RFC, you mentioned that it's the version of the RFC. Could you please clarify its meaning?

the process RFC describes it as XCM version with which the proposal will be enacted.
may be worth to have both, XCM Version, and RFC Version.
with RFC version, we could leave a room for such minor changes.
I think it's not that nice that this RFC changes it's number and the location / file name.

File: proposals/0032-process.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 Title: RFC process
-Number: 1
+Number: 32
```
[the remark this answers is not in the document]
Comment (KiChjang) [reply proposals/0032-process.md:3]: Because the original PR that introduced this RFC is #32.

**Comment by KiChjang** [comment]
I don't think we should have RFC versioning. Naturally, the instinct is to look for older versions of the same RFC, but those are irrelevant as long as we make sure the newer RFC text doesn't conflict with the old one.

**Pull request merged**